### PR TITLE
ci(travis): Publish nightlies to getsentry/sentry:git

### PR DESCRIPTION
**Pull request by BYK**
Publishes the git builds to
[getsentry/sentry:git](https://hub.docker.com/r/getsentry/sentry)

### CODE REVIEW

File: git/onbuild/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM sentry:git
+FROM getsentry/sentry:git
```
Comment (BYK) [inline git/onbuild/Dockerfile]: I can rename this to `getsentry/sentry:nightly` to align better with the common names.

File: .travis.yml [hunk 2]
```diff
@@ -18,9 +18,15 @@ script:
...
   - $TRAVIS_BUILD_DIR/test/run.sh "$image"
-  - if [ -f onbuild/Dockerfile ]; then docker build -t "$image-onbuild" onbuild; fi
-  - if [ -f onbuild/Dockerfile ]; then ~/official-images/test/run.sh "$image-onbuild"; fi
-  - if [ -f onbuild/Dockerfile ]; then $TRAVIS_BUILD_DIR/test/run.sh "$image-onbuild"; fi
+  - docker build -t "$image-onbuild" onbuild
+  - ~/official-images/test/run.sh "$image-onbuild"
+  - $TRAVIS_BUILD_DIR/test/run.sh "$image-onbuild"
+  - >
+    if [ "$VERSION" == "git" -a "$TRAVIS_PULL_REQUEST" == "false" -a "$TRAVIS_BRANCH" == "master" ]; then
+      echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin &&
```
Comment (mattrobenolt) [inline .travis.yml]: Where are these values coming from? And what’s preventing them from being leaked?

Comment (BYK) [reply .travis.yml]: They are set in Travis config. For leaks: https://docs.travis-ci.com/user/best-practices-security#recommendations-on-how-to-avoid-leaking-secrets-to-build-logs

File: git/onbuild/Dockerfile (hunk 1 above)
Comment (mattrobenolt) [reply git/onbuild/Dockerfile]: I don't havea  strong preference, we can always change later. It'd be nice if we ultimately were tagging with some sort of timestamp or sha. Again, can be addressed later.

**Pull request merged**